lms/static/sass/shared/_footer-edx.scss

```diff
@@ -7,17 +7,116 @@
 $edx-footer-link-color: $link-color;
 $edx-footer-bg-color: rgb(252,252,252);
 
-
 // Aggressively scoped for Drupal
 // ==============================
 // These styles are being loaded on Drupal, LMS and WordPress
 // sites so the scope has to be aggressive to override default
 // CMS styles
 footer#footer-edx-v3 {
   background: $edx-footer-bg-color;
-  padding: 20px;
+  padding: 15px 0;
   border-top: 1px solid $courseware-button-border-color;
+  font-family: "Open Sans" ;
```

Comment:
Why is this line necessary. It should just use the default font family which lists "Open Sans" first.